pyk/src/pyk/kcfg/kcfg.py

```diff
@@ -1234,9 +1237,6 @@ def write_cfg_data(self) -> None:
         self._deleted_nodes.clear()
         self._created_nodes.clear()
 
-    def minimize(self) -> None:
-        KCFGMinimizer(self).minimize()
```

Comment:
If this is deleted, Kontrol will no longer function as before since it calls `KCFG.minimize()`.

Reply:
However, I haven’t figured out how to break this circular import path without modifying the existing interface and functionality for the downstream repos. Does this circular import cause any issues? I didn’t notice any errors during my testing.

Reply:
It does cause more issues as things are added to th efile. Circular imports are always a problem.

I intend to fix the downstream repos, here is the fix for Kontrol: https://github.com/runtimeverification/kontrol/pull/793/commits/dfd7222f28fb857fd010779e76d37443bb124149

Reply:
Thank you! I think this is a good way.

Reply:
Consider:

```python3
def minimize(self) -> None:
    from .minimize import KCFGMinimizer
    KCFGMinimizer(self).minimize()
```

This way, the interface can be maintained. This (and the relative import fix) can be pulled out in a separate PR and merged right away.

Reply:
I'm inclined to leave it as is, as I'll be pushing through the updates today anyway. But can change it if you like.